src/PrestaShopBundle/Controller/Admin/CommonController.php

```diff
@@ -42,6 +42,33 @@
  */
 class CommonController extends FrameworkBundleAdminController
 {
+    /**
+     * Get a summary of recent events on the shop.
+     * This includes:
+     * - Created orders
+     * - Registered customers
+     * - New messages.
+     *
+     * @return JsonResponse
+     */
+    public function notificationsAction()
+    {
+        // Adapter needed here?
+        return new JsonResponse((new \Notification())->getLastElements());
+    }
+
+    /**
+     * Update the last time a notification type has been seen.
+     *
+     * @param Request $request
+     */
+    public function notificationsAckAction(Request $request)
+    {
+        $type = $request->request->get('type');
+        // Adapter needed here?
```

Comment:
Ping @mickaelandrieu @matks 
I think an adapter is needed

Reply:
I'll create an issue in Migration kanban for this one. Let's not fix it in this PR.

Reply:
Why us 😛 everybody can write an adapter